src/SFA.DAS.PR.Application/Notifications/Commands/PostNotificationsCommandResult.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.PR.Application.Notifications.Commands;
+
+[ExcludeFromCodeCoverage]
+public record PostNotificationsCommandResult();
```

Comment:
Mediator has something called `Unit` when  there is nothing to return like in case of commands, the return type can be `Unit` instead of empty object.